KachnaOnline.Business/Services/StatePlanning/TransitionHandlers/PushTransitionHandler.cs

```diff
@@ -0,0 +1,38 @@
+// PushTransitionHandler.cs
+// Author: František Nečas
+
+using System.Threading.Tasks;
+using KachnaOnline.Business.Configuration;
+using KachnaOnline.Business.Services.Abstractions;
+using KachnaOnline.Business.Services.Push;
+using KachnaOnline.Business.Services.StatePlanning.Abstractions;
+using Lib.Net.Http.WebPush;
+using Microsoft.Extensions.Options;
+
+namespace KachnaOnline.Business.Services.StatePlanning.TransitionHandlers
+{
+    public class PushTransitionHandler : PushNotificationClient, IStateTransitionHandler
+    {
+        private readonly IPushSubscriptionsService _pushSubscriptionsService;
+
+        public PushTransitionHandler(IOptionsMonitor<PushOptions> pushOptions, PushServiceClient pushClient,
+            IPushSubscriptionsService pushSubscriptionsService) : base(pushOptions, pushClient)
+        {
+            _pushSubscriptionsService = pushSubscriptionsService;
+        }
+
+        public async Task PerformStartAction(int stateId, int? previousStateId)
+        {
+            // TODO: add a better message.
```

Comment:
The proof is trivial and was left to the reader as an exercise, something for you @ondryaso . Feel free to push into my branch